src/gui/accelerators.c

```diff
@@ -192,7 +192,7 @@ void dt_accel_register_lib(dt_lib_module_t *self, const gchar *path, guint accel
     else if(strcmp(views[i], "*") == 0)
       accel->views |= DT_VIEW_DARKROOM | DT_VIEW_LIGHTTABLE | DT_VIEW_TETHERING | DT_VIEW_MAP | DT_VIEW_PRINT
                       | DT_VIEW_SLIDESHOW;
-    i++;  
+    i++;
```

Comment:
Not important, but could have been omitted from this PR. All formatting are adding more things to review.

Reply:
Sorry, I'll add this space back.

Reply:
No, no need :) It was a comment for next PR.

Reply:
Sorry ... too late ...

Reply:
ok, understood !